lib/core/terraform_config/required_provider.rb

```diff
@@ -13,6 +13,10 @@ def initialize(name, **options)
 
     def to_tf
       block :terraform do
+        block :cloud do
+          argument :organization, "PLACEHOLDER"
```

Comment:
I'm not really sure what this means, can we not use the value of `config.org`? Or is this something else?

I'll leave it up to you and @borela, since I'm not too familiar with terraform.

Reply:
@rafaelgomesxyz good point, @zzaakiirr that option mentioned by Rafael makes sense in this context.  

@rafaelgomesxyz Terraform uses workspaces to store the resources deployed info, those workspaces will get stored in a terraform cloud account and we have to specify the org to access those workspaces.

Reply:
Done